imports/plugins/included/marketplace/client/templates/shops/shopSelect.js

```diff
@@ -0,0 +1,71 @@
+import { Meteor } from "meteor/meteor";
+import { Template } from "meteor/templating";
+import { ReactiveVar } from "meteor/reactive-var";
+import { Reaction } from "/lib/api";
+import { SellerShops } from "/lib/collections";
+
+Template.shopSelect.onCreated(function () {
+  this.autorun(() => {
+    // watch path change to reset toggle
+    Reaction.Router.watchPathChange();
+  });
+});
+
+Template.shopSelect.helpers({
+  sellerShops() {
+      const currentShopId = Reaction.Router.getParam("shopId") || 0;
+      const selector = {
+        // ignore blank site
+        _id: {
+          $ne: "ddzuN2YPvgvx7rJS5"
```

Comment:
We have be updated this publication with observers now, which allow for transforms in seller data and are no longer ignoring any id. Will create a PR by tomorrow with that, and other core changes.